pallets/gear/src/benchmarking/code.rs

```diff
@@ -315,7 +328,24 @@ where
             )));
         }
 
-        let code = program.build();
+        let mut code = program.build();
+
+        // Add dummy type section
+        if let Some(types) = def.types {
+            for section in code.sections_mut() {
+                if let Section::Type(sec) = section {
+                    for _ in 0..types.num_elements {
+                        sec.types_mut().push(Type::Function(FunctionType::new(
```

Comment:
pretty interesting that its still valid wasm module

Reply:
spec doesn't seem to forbid duplicated and unused types, so it seems logical.

Reply:
yeah, sure, I was just sharing some thoughts)